app/fetch-did-web-doc.py

```diff
@@ -26,8 +27,11 @@ def fetch_unattested(url, nonce):
         {
             "url": url,
             "nonce": nonce,
-            "status_code": response["status_code"],
-            "body": base64.b64encode(response["body"]).decode("utf-8"),
+            "result": {
+                "status": response["status_code"],
+                "body": base64.b64encode(response["body"]).decode("utf-8"),
+                "cert_chain": [],
+            },
```

Comment:
This is going to break the unattested parser in the ledger, and also doesn't work with refetchable. Given that `request` already does retries on http errors, there's no need to try and unify the two code paths together at this point. We can do that when we have the "virtual" attested-fetch, and then we'll be able to simplify this file.

Reply:
Reverted the unattested changes